packages/cln_plugin/lib/src/json_rpc/error.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'error.g.dart';
+
+@JsonSerializable()
+class Error {
+  int code;
+  String message;
+  dynamic data;
```

Comment:
Review needed, for the time being I have added it as a `dynamic` but the spec dictates `A Primitive or Structured value that contains additional information about the error. This may be omitted. The value of this member is defined by the Server (e.g. detailed error information, nested errors etc.).`